corehq/apps/app_manager/exceptions.py

```diff
@@ -204,3 +204,10 @@ class MobileUCRTooLargeException(CannotRestoreException):
     def __init__(self, message, row_count):
         super().__init__(message)
         self.row_count = row_count
+
+
+class AppInDifferentDomainException(AppManagerException):
+    """
+    This exception is raised when an app is requested from a domain that is different from the domain of the app.
```

Comment:
This docstring references two different apps. Which is the second app referring to? Seems like there is some missing context.

Reply:
How about update the docstring to be "This exception is raised when an app is not in the targeted domain" ?